visallo/visallo
# update find message on welcome to visallo dashboard card to reflect that you can search both entities and relationships

Testing Instructions: 
On Visallo Welcome dashboard card, the message under find should be `To find entities or relationships in Visallo use the Find button.`

Points of Regression: N/A

CHANGELOG
Changed:  Find description on Visallo Dashboard card to reflect that you can search entities or relationships
